CONTRIBUTING.md

```diff
@@ -38,11 +38,20 @@ Clone the repository to your local machine from the Azure DevOps endpoint.
 
 - Using Powershell, navigate to the '/RetrospectiveExtension.Frontend' folder, run `npm install`. This will download all the dependent packages listed in 'package.json'.
 
+- Copy the file `RetrospectiveExtension.Frontend\config\environment.tsx.template` into `RetrospectiveExtension.Frontend\config\environment.tsx` and update the fields
+
+```json
+{
+   "CollaborationStateServiceUrl": "https://my-backend-service.com", // change this to the deployed backend service
+   "AppInsightsInstrumentKey": "my_instrumentation_key" // put Instrumentation key here
+}
+```
+
 - Run `npm run build` to build the project. Refer to the 'scripts' section in 'package.json' for other commands.
 
 - To test your changes, you will need to publish a new extension under a new Azure DevOps publisher account. Refer to the [documentation](https://docs.microsoft.com/en-us/azure/devops/extend/publish/overview?view=vsts) on publishing extensions. You can publish it to any test Azure DevOps organization that you are an admin of (As a Microsoft employee, you can create a new test organization from your Azure DevOps profile page). Currently this is the only way to test the extension.
 
-- Update the 'vss-extension-dev.json' file with the new publisher that you setup. Also update the name and id fields.
+- Copy the file `vss-extensions-dev.json.template` into `vss-extensions-dev.json` file with the new publisher that you setup. Also update the name and id fields.
```

Comment:
`vss-extension-dev.json` (singular) file ignored in the `.gitignore` file, here we're stating the file as plural `(vss-extensions-dev.json)`

either of them should be changed I think.

Reply:
changed file name to `vss-extension-dev` singular form